tensorboard/webapp/notification_center/_redux/notification_center_reducers.ts

```diff
@@ -45,6 +46,21 @@ const reducer = createReducer(
         lastReadTimestampInMs: timeNow,
       };
     }
+  ),
+  on(
+    actions.fetchNotificationsLoaded,
+    (
+      state: NotificationState,
+      response: NotificationCenterResponse
+    ): NotificationState => {
+      if (response.notifications) {
+        return {
+          ...state,
+          notifications: response.notifications!,
```

Comment:
`response.notifications!`: because you already checked for its existence in L56, you don't need this non-nullable coercion; TypeScript already refine d the type.